package/obs/rmt.conf

```diff
@@ -11,6 +11,7 @@ database:
 scc:
   username:
   password:
+  sync_systems: true
```

Comment:
I think we should change the default `config/rmt.yml` as well to `sync_systems: false`

Reply:
It's the same default behavior as SMT. Ultimately, that's up to PM to decide -- so feel free to check with Benjamin.

Reply:
that is correct, I had a talk with @BLandorff and we agreed that it has to be default behavior to sync systems. but then we have another thing: systems with installed RMT will not get config file updated and will not sync systems. maybe it's better to have this setting opt-out - this means is it is not defined in config file we will sync systems by default?

Reply:
Systems receiving the update won't have the new systemd timers enabled regardless of whether it's opt-in or opt-out in the config, so that doesn't really matter.